run-tests-with-build-drools-parent/drools-project-sources-test/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>run-tests-with-build-drools-parent</artifactId>
+    <groupId>org.kie</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <artifactId>drools-project-sources-test</artifactId>
+
+  <properties>
+    <filter1>drools-persistence/drools-persistence-jpa</filter1>
+    <filter2>drools-persistence/drools-persistence-api</filter2>
+    <filter3>drools-test-coverage/test-integration-nomvel</filter3>
+    <filter4>drools-impact-analysis/drools-impact-analysis-itests</filter4>
+    <filter5>drools-impact-analysis/drools-impact-analysis-parser</filter5>
+    <filter6>drools-impact-analysis/drools-impact-analysis-model</filter6>
+    <filter7>drools-impact-analysis/drools-impact-analysis-graph/drools-impact-analysis-graph-common</filter7>
+    <filter8>drools-impact-analysis/drools-impact-analysis-graph/drools-impact-analysis-graph-graphviz</filter8>
```

Comment:
One filter per row to avoid make maven.modules.explicit.filtering row too long to improve readability.